test/e2e/virtual/syncer/virtualworkspace_test.go

```diff
@@ -1965,6 +1965,19 @@ func TestUpsyncerVirtualWorkspace(t *testing.T) {
 				return true
 			}, wait.ForeverTestTimeout, time.Millisecond*100)
 
+			kubelikeUpsyncerVWClient, err := kcpkubernetesclientset.NewForConfig(upsyncer.UpsyncerVirtualWorkspaceConfig)
+			require.NoError(t, err)
+
+			logWithTimestampf(t, "Waiting for the persistentvolumes to be available in the upsyncer virtual workspace...")
+			require.Eventually(t, func() bool {
```

Comment:
Please use framework.Eventually - thanks!

Reply:
Oh, yes, copy/paste obviously, Sorry. I'll change that immediately !

Reply:
Fixed and rebased.